src/components/ProgramInfo.tsx

```diff
@@ -0,0 +1,88 @@
+import { Flex, Heading, Spacer, Badge, Text } from "@chakra-ui/react";
+import Wrapper from "@components/SDCWrapper";
+import { BackButton } from "@components/BackButton";
+import { Button } from "@chakra-ui/react";
+import { ClassList } from "src/components/ClassList";
+import type { ClassCardInfo } from "models/Class";
+
+interface ProgramDetailsProps {
+    styleProps?: Record<string, unknown>;
+    programInfo: ProgramInfoType;
+    session: Record<string, unknown>;
+    classInfo: ClassCardInfo[];
+}
+
+type ProgramInfoType = {
```

Comment:
thoughts on moving this to a different file ? @feng-eric

Reply:
@mahadzaryab1 Do you mean like `models/Program.ts`? I'm ok with that yeah

Reply:
I just noticed that this exact type already exists in `models/Program.ts` :(